plotting/plot_factory/dashapp.py

```diff
@@ -0,0 +1,43 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+Constructs a plot and DashApp object for insertion into directly into a web page
+"""
+
+# Core Dependencies
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+
+
+class DashApp:  # pylint: disable=too-few-public-methods
+    """Creates a DashApp for direct insertion into a web page"""
+    def __init__(self, figure, app_id):
+        """
+        Dashapp object properties
+        :param figure (dictionary)
+        :param app_id (string)
+        """
+        self.figure = figure
+        self.app_id = app_id
+        self.app = self.create_dashapp()
+
+    def create_dashapp(self):  # pylint: disable=too-few-public-methods
```

Comment:
Is this `#pylint:disable` required here? If you shift the one on line 17 to be on line 16, I believe you can then remove this one.
From memory if you do inline pylint disables it will work only for that line but we want this one to operate for the scope of the class. Moving the pylint disable to be abovethe class definition means it will then operate of the class scope